src/backend/app/projects/project_schemas.py

```diff
@@ -278,20 +278,27 @@ async def one(db: Connection, project_id: uuid.UUID):
 
     async def all(
         db: Connection,
+        user_id: Optional[uuid.UUID] = None,
```

Comment:
is the user id UUID? If not, better to have it `str`

Reply:
@nrjadkry  I checked and found that `user_id` is stored as `str`. I've updated the code to reflect this.